app/views/tutors/show.html.erb

```diff
@@ -358,6 +358,16 @@
 
     <hr>
 
+    <!-- ================================================================= -->
+    <div class="row">
+        <h4 class="col-md-12"><strong>Affiliate</strong></h4>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <strong>Affiliate:</strong><br>
+        <%= @tutor.affiliate_id %>
```

Comment:
No, a foreign key still makes sense. But seeing "Affiliate: 3" on the page won't be helpful to the user. Something like `Affiliate.find(@tutor.affiliate_id).name` would make more sense here.